roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -56,6 +56,12 @@
     l_common_version: "{{ common_version.stdout | default('0.0', True) }}"
   when: not openshift.common.is_containerized | bool
 
+# This can be removed when OSE 3.0 is no longer supported.
+- name: Set docker version to be installed
+  set_fact:
+    docker_version: "{{ '1.8.2' }}"
+  when: " ( l_common_version | version_compare('3.2','<') and openshift.common.service_type == 'openshift' )"
```

Comment:
Online should already be covered since we're talking about service_types here.  I did update the PR based on the change we discussed before scrum.